### PR TITLE
Update UserCase ID

### PR DESCRIPTION
**Pull request by maxyzx**
### Description
- Add ID for UserCase payload (As Block in Jitera Core)

### Version
- Node Proto Version: 0.0.2-beta.70
- Ruby Proto Version: 0.0.52

### Issues
- Part of https://jitera.atlassian.net/browse/JSN-285

### CODE REVIEW

File: protos/v2/user_case_creation.proto [hunk 1]
```diff
@@ -21,14 +21,15 @@ message UserCaseCreation {
...
-  string name = 1;
-  string node_id = 2;
-  string parent_node_id = 3;
-  string block_type = 4;
-  repeated Content content = 5;
-  Property properties = 6;
-  int32 project_id = 7;
-	repeated UserCase children = 8;
+	int32 id = 1;
+  string name = 2;
```
Comment (zrg-team) [inline protos/v2/user_case_creation.proto:25]: is there problem with format ?

**Review by zrg-team — CHANGES_REQUESTED** [review]

File: protos/v2/user_case_creation.proto (hunk 1 above)
Comment (maxyzx) [reply protos/v2/user_case_creation.proto:25]: There is no problem. I don't know why github using different Tab size with local editor

Comment (akuita) [reply protos/v2/user_case_creation.proto:25]: why not use the same indentation? spacing in this case

Comment (maxyzx) [reply protos/v2/user_case_creation.proto:25]: @zrg-team  @akuita It looks fine in VScode. 

image

**Review by zrg-team — APPROVED** [review]

**Pull request merged**